cases/toml.test.js

```diff
@@ -8,13 +8,21 @@ const url = urlBuilder.toString();
 
 describe("TOML File", () => {
   it("exists", async () => {
-    const response = await fetch(url + ".well-known/stellar.toml");
-    expect(response.status).toBe(200);
+    const { status, logs } = await loggableFetch(
+      url + ".well-known/stellar.toml",
+      {},
```

Comment:
Not really clean to send the "{}". How can I use the default value from loggableFetch?